ALPSCore Master does not compile in DEBUG mode

When I try to build ALPSCore in debug mode I get the following compile-time error:

      Linking CXX executable params_origins
      .......
      Undefined symbols for architecture x86_64:
        "std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::c_str() const", referenced from:
            arg_holder::argv() in params_origins.cpp.o
      ld: symbol(s) not found for architecture x86_64

--- a/params/test/params_test_support.hpp
+++ b/params/test/params_test_support.hpp
@@ -5,7 +5,7 @@
  */
 
 /** @file params_test_support.hpp
-    
+
     @brief Utility classes to support param testing
 */
 
@@ -29,7 +29,7 @@ class arg_holder {
     explicit arg_holder(const std::string progname="./program_name") {
         this->add(progname);
     }
-    
+
     arg_holder& add(const std::string& arg) {
         args_.push_back(arg);
         return *this;
@@ -43,8 +43,8 @@ class arg_holder {
     const char* const * argv() {
         argv_.resize(args_.size());
         std::transform(args_.begin(), args_.end(), argv_.begin(),
-                       std::const_mem_fun_ref_t<const char*, std::string>(&std::string::c_str));
-        return &argv_.front();
+                       [](const std::string& s) { return s.c_str(); });
+        return argv_.data();
     }
 };
 

